src/frontend/src/pages/Auth/Login.tsx

```diff
@@ -94,22 +96,28 @@ export default function Login() {
       ) : (
         <>
           <Wrapper titleText={t`Login`} smallPadding>
-            <AuthenticationForm />
-            {both_reg_enabled === false && (
-              <Text ta='center' size={'xs'} mt={'md'}>
-                <Trans>Don&apos;t have an account?</Trans>{' '}
-                <Anchor
-                  component='button'
-                  type='button'
-                  c='dimmed'
-                  size='xs'
-                  onClick={() => navigate('/register')}
-                >
-                  <Trans>Register</Trans>
-                </Anchor>
-              </Text>
+            {isLoggingIn ? (
+              <Loader />
+            ) : (
+              <>
+                <AuthenticationForm />
+                {both_reg_enabled === false && (
+                  <Text ta='center' size={'xs'} mt={'md'}>
+                    <Trans>Don&apos;t have an account?</Trans>{' '}
```

Comment:
Do quote characters not get properly escaped inside the `<Trans>` block?

Reply:
Not really sure, this was autoformatted in the prior PR and seems to render fine

Reply:
If possible I would prefer the literal `Don't` here - it will make the translator's job easier?

Reply:
Ok, fair enough